tests/0.8.1/invalid.json

```diff
@@ -0,0 +1,842 @@
+[
+  {
+    "comment": "UCAN sections contain invalid base64 characters",
+    "token": "@@JhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOCJ9.eyJpc3MiOiJkaWQ6a2V5Ono2TWtySnpWZVk3RXd2cVdNc3ROeDJ4SDdiZGVnYWREZGFjOHVGams2NTlnemJkeCIsImF1ZCI6ImRpZDprZXk6ek0wMG04RHhXU3dRaGhaWWJnUGprQ05qbUx2dmEzRDdxQnNHUHZ3ejJneW5TaWFKIiwiZXhwIjoxNjQ4MDc2MDQ0LCJhdHQiOltdLCJwcmYiOltdfQ.vLgf-O3Xrc94toabVJKoI15EmAO-d9SqHGFbxTkw3wyEZb-Q7YXA-6mWJv6b-b7kFQHiilMcdIl5s5IuXxsSAw",
+    "assertions": {
+      "validationErrors": [
+        "base64Invalid"
+      ]
+    }
+  },
+  {
+    "comment": "UCAN header section is missing or invalid",
```

Comment:
We might want to say "UCAN header section is missing" and add a separate test case for an invalid header.

Reply:
Got further down into the invalid cases, and I noticed there are cases of headers with fields missing. That's what I would consider and "invalid" header. No need to add more test cases (same goes for payload comment below, but we could use a garbage signature test case.)

Reply:
The reason for having it as `missing or invalid` here is because it can be non-trivial to detect the intention of the UCAN author. For example, the header may in fact not be missing but malformed maybe by having some additional non-compliant fields or missing some compliant fields. If it just says `missing` then we are forcing the implementer to detect all of those cases.
I can change the wording to `malformed`.

Reply:
Good point. Reading through the example, it was trivial to see that the header was missing, but testing for it would be challenging. Renaming to `malformed` sounds good to me. 👍